docs/releases/v2023.1.rst

```diff
@@ -0,0 +1,121 @@
+Gluon 2022.1
+============
+
+Important notes
+---------------
+
+Upgrades to v2022.1 and later releases are only supported from releases v2021.1 and later. This is due to migrations that have been removed to simplify maintenance.
+
+
+Added hardware support
+----------------------
+
+ath79-generic
+~~~~~~~~~~~~~
+
+- Extreme Networks
+
+  - WS-AP3805i
+
+ath79-nand
+~~~~~~~~~~
+
+- GL.iNet
+
+  - GL-XE300
+
+ramips-mt7621
+~~~~~~~~~~~~~
+
+- TP-Link
+
+  - EAP615-Wall
+
+- Wavlink
+
+  - WS-WN572HP3 4G
+
+ramips-mt76x8
+~~~~~~~~~~~~~
+
+- TP-Link
+
+  - TL-MR6400 (v5)
+
+
+Features
+--------
+
+DNS-Caching
+~~~~~~~~~~~
+
+DNS-Caching using the dnsmasq resolver is reintroduced. This requires configuring the
+caching parameters. See the :ref:`DNS-Caching documentation <dns-caching>` section for
```

Comment:
"This requires configuring the caching parameters." might be misunderstood - it is not clear that the configuration is only required to use the caching.

Maybe the link to the doc page is sufficient, and the sentence can just be dropped?